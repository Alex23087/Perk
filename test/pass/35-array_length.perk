import "stdio.h"

fun main(): void {
    let arr: [int] = [5, 4, 3, 2, 1]; // Initialize an array with values
    printf("Array elements: ");
    for (let i: int = 0; i < arr.length; i++) {
        if (i > 0) {
            printf(", ");
        };
        printf("%d", arr[i]);
    }
}