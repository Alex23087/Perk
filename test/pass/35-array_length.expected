Array elements: 5, 4, 3, 2, 1
